Gems/ROS2/Code/Source/RobotImporter/xacro/XacroUtils.cpp

```diff
@@ -7,10 +7,13 @@
  */
 
 #include "XacroUtils.h"
+#include <QString>
+#include <AzCore/Settings/SettingsRegistryMergeUtils.h>
 #include <AzCore/IO/FileIO.h>
 #include <AzCore/XML/rapidxml.h>
 #include <AzFramework/Process/ProcessCommunicator.h>
 #include <AzFramework/Process/ProcessWatcher.h>
+#include <QString>
```

Comment:
QString included twice. Also, this seems not relevant to the lidar PR

Reply:
Opened issue for changes in this file https://github.com/o3de/o3de-extras/issues/122